CLI/Repl.cpp

```diff
@@ -169,7 +169,17 @@ static std::string runCode(lua_State* L, const std::string& source)
     }
     else
     {
-        std::string error = (status == LUA_YIELD) ? "thread yielded unexpectedly" : lua_tostring(T, -1);
+        std::string error;
+
+        if (status == LUA_YIELD)
+        {
+            error = "thread yielded unexpectedly";
+        }
+        else if (const char* str = lua_tostring(L, -1))
```

Comment:
@zeux Seems like this slipped through. In this context `T` represents the Luau VM context.

Reply:
Single-letter variable names ftw. Thanks!

Reply:
Fixed in master